compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -429,21 +433,45 @@ internal class RootNodeOwner(
             drawBlock: (canvas: Canvas, parentLayer: GraphicsLayer?) -> Unit,
             invalidateParentLayer: () -> Unit,
             explicitLayer: GraphicsLayer?,
-        ) = RenderNodeLayer(
-            density = Snapshot.withoutReadObservation {
-                // density is a mutable state that is observed whenever layer is created. the layer
-                // is updated manually on draw, so not observing the density changes here helps with
-                // performance in layout.
-                density
-            },
-            measureDrawBounds = platformContext.measureDrawLayerBounds,
-            invalidateParentLayer = {
-                invalidateParentLayer()
-                snapshotInvalidationTracker.requestDraw()
-            },
-            drawBlock = drawBlock,
-            onDestroy = { needClearObservations = true }
-        )
+        ) = if (explicitLayer != null) {
+                GraphicsLayerOwnerLayer(
+                    graphicsLayer = explicitLayer,
+                    context = null,
+                    drawBlock = drawBlock,
+                    invalidateParentLayer = {
+                        invalidateParentLayer()
+                        snapshotInvalidationTracker.requestDraw()
+                    },
+                )
+            } else {
+                /*
+                TODO: Use GraphicsLayerOwnerLayer instead of RenderNodeLayer
+                GraphicsLayerOwnerLayer(
```

Comment:
If we uncomment this, do `GraphicsLayerTest` and `RenderNodeLayerTest` pass? If not, it may be an indicator that `GraphicsLayerOwnerLayer` isn't correctly implemented

Reply:
The only problem I am aware of is a lack of picture measurement support

Reply:
I see 2 failed tests:
- box outside bounds should not be clipped. Can we apply the same fix with PICTURE_BOUNDS?
- elevation. it looks different.

Reply:
1. tried to do that, other test fail

Reply:
Valid points for the future, but we don't need to fix all bugs at once

Reply:
https://youtrack.jetbrains.com/issue/CMP-6616/GraphicLayer.-Fix-elevation
https://youtrack.jetbrains.com/issue/CMP-6617/GraphicLayer.-Fix-clipping